src/py/mat3ra/made/material.py

```diff
@@ -108,3 +108,8 @@ def set_new_lattice_vectors(
         self.basis = new_basis
         lattice = Lattice.from_vectors_array([lattice_vector1, lattice_vector2, lattice_vector3])
         self.lattice = lattice
+
+    def add_atom(self, element: str, coordinate: List[float]) -> None:
```

Comment:
What if this atom intersects with others?

Reply:
Default coordinate could be zero

Reply:
Check-in basis with force=False

Reply:
Still needs to be resolved

Reply:
Yes. This was added in `src/py/mat3ra/made/basis.py` https://github.com/Exabyte-io/made/pull/154/files#diff-b8cd5b3322adff253e8773f21fd3c8c1919394b0cdc166c1ac4357c705293f32
Since we don't have Periodic Table in Python with their covalent radii, `add_atom` just tests to collisions with set tolerance: 0.01 A by default.